src/components/Blockquote/Blockquote.docs.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Preview } from '@storybook/addon-docs/blocks';
+import Blockquote from './Blockquote';
+
+# Blockquote
+
+A stylized blockquote that accentuates a quote from a larger body of text. Renders an HTML `<blockquote>` tag.
+
+## Usage
+
+```js
+import { Blockquote } from '@quartz/interface/src/components';
+```
+
+## Props
+
+<Props of={Blockquote} />
+
+## Usage Guidelines
+- **Do** wrap a `Blockquote` component to change the text style or constrain its width.
+- **Do not** use the `Blockquote` component if the text is not a direct quote from a person or source material.
+
```

Comment:
May also want to add something about placement of the quote citation, I've noticed that sometimes we put them inside the Blockquote and sometimes outside (I think I prefer inside)

Reply:
That's a good point/question. We have them outside for the testimonials in the subscribe flow, but I agree that in articles with larger quote blocks, it looks better to have the citation in the block. Maybe a Q for Daniel or Noah?

Reply:
That's up to the editor in WP, maybe better to ask someone on the edit team what they prefer and follow their lead